h/views/admin/groups.py

```diff
@@ -162,7 +160,7 @@ def __init__(self, request):
         user_svc = request.find_service(name="user")
         self.request = request
         self.schema = CreateAdminGroupSchema().bind(
-            request=request,
+            request=self.request,
```

Comment:
There's a fair bit of messiness in these classes still regarding when things are set as instance attrs and when they're just vars within a given method. Would like to clean up.